README.md

```diff
@@ -7,10 +7,20 @@ This can especially be helpful for addon creators, server owners and people who
 This can be helpful with testing how optimized the blocks are or if they would cause any issues in mass usage.
 
 ## Commands
-- `/wesf pos1` This command sets the location for position 1 for your paste/clear command.
-- `/wesf pos2` This command sets the location for position 2 for your paste/clear command.
-- `/wesf paste <Slimefun ID>` This command pastes the block you specified in the world with the position you chose. It has 1 argument that is the slimefun item id. This can be any placeable Slimefun or Slimefun Addon block.
-- `/wesf clear <boolean>` This command clears the blocks you have selected with the position commands. When `true` it fires a blockbreakevent coming from a player. This is used to clear all the handlers. When `false` it just removes the block and the blockstorage data.
+- `/wesf wand` 
+  - This command gives a player the selection wand
+- `/wesf pos1`
+  - This command sets position 1 of your selection.
+- `/wesf pos2` 
+  - This command sets position 2 of your selection.
+- `/wesf paste <slimefun_block> <energy> <inputs>` 
```

Comment:
```suggestion
- `/wesf paste <slimefun_block> [energy] [inputs]` 
```

Reply:
going to switch to Jeff's idea, so this is going to change regardless